clique/src/mock.rs

```diff
@@ -667,3 +667,12 @@ pub struct MockPrelims<D> {
     pub result_from_outgoing: UnboundedReceiver<ResultForService<MockPublicKey, D>>,
     pub authorization_requests: mpsc::UnboundedReceiver<(MockPublicKey, oneshot::Sender<bool>)>,
 }
+
+pub struct MockMetrics {}
```

Comment:
terrible nit to use less characters :p
```suggestion
pub struct MockMetrics;
```

Reply:
Renamed to `NoopMetrics` and moved to `metrics.rs` file so that it can be used as a default implementation also for non-testing code.